db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -552,8 +552,18 @@ class NonBatchedOpsStressTest : public StressTest {
     }
     shared->Put(rand_column_family, rand_key, value_base, false /* pending */);
     if (!s.ok()) {
-      fprintf(stderr, "put or merge error: %s\n", s.ToString().c_str());
-      std::terminate();
+      if (FLAGS_injest_error_severity >= 2) {
+        if (!is_db_stopped_ && s.severity() >= Status::Severity::kFatalError) {
+          is_db_stopped_ = true;
+        } else if (!is_db_stopped_ ||
+                   s.severity() < Status::Severity::kFatalError) {
+          fprintf(stderr, "put or merge error: %s\n", s.ToString().c_str());
```

Comment:
If its a retryable error, we always expect the Put/Delete to succeed. Is that because the fault injection FS never injects errors on WAL writes?

Reply:
> If its a retryable error, we always expect the Put/Delete to succeed. Is that because the fault injection FS never injects errors on WAL writes?

Yes, in the current fault injection, we set for SST, Manifest, and Current. https://github.com/facebook/rocksdb/pull/8479#discussion_r662412921